docs/user-guide/rules.md

```diff
@@ -141,6 +141,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`selector-combinator-space-after`](../../src/rules/selector-combinator-space-after/README.md): Require a single space or disallow whitespace after the combinators of selectors.
 - [`selector-combinator-space-before`](../../src/rules/selector-combinator-space-before/README.md): Require a single space or disallow whitespace before the combinators of selectors.
 - [`selector-id-pattern`](../../src/rules/selector-id-pattern/README.md): Specify a pattern for id selectors.
+- [`selector-max-depth`](../../src/rules/selector-max-depth/README.md): Limit the depth of selectors.
```

Comment:
> This rule counts compound selectors, right

Tunrs out that - yes, now it would count compound selectors. So `selector-max-combinators` for simplicity (instead of `selector-max-compound-selectors`)?